frontend/apps/crates/entry/admin/src/curation/table/dom.rs

```diff
@@ -74,77 +83,123 @@ impl CurationTable {
+                                    }
+                                }))
+                                .event(clone!(state => move |_: events::Click| {
+                                    let mut blocked = jig.blocked.lock_mut();
+                                    *blocked = !*blocked;
+                                }))
+                            }))
+                        }),
                         html!("span", {
-                            .text(&jig.author_name.unwrap_or_default())
+                            .text(&jig.author_name)
                         }),
                         html!("star-rating", {
-                            .property("rating", jig.admin_data.rating.map(|rating| {
-                                rating as u8
+                            .property_signal("rating", jig.rating.signal().map(|rating| {
+                                match rating {
+                                    Some(rating) => rating as u8,
+                                    None => 0,
+                                }
                             }))
                         }),
-                        // html!("span", {
-                        //     .text("AUTHOR BADGE")
-                        // }),
                         html!("span", {
                             .text(&match jig.published_at {
                                 Some(published_at) => published_at.format("%b %e, %Y").to_string(),
                                 None => "".to_string()
                             })
                         }),
                         html!("span", {
-                            .text(Language::code_to_display_name(&jig.jig_data.language))
+                            .text_signal(jig.language.signal_cloned().map(|language| {
+                                Language::code_to_display_name(&language)
+                            }))
                         }),
-                        // html!("span", {
-                        //     .text("CURATORS")
-                        // }),
                         html!("span", {
                             .style("display", "flex")
                             .style("flex-wrap", "wrap")
                             .style("column-gap", "16px")
-                            .children(jig.jig_data.age_ranges.into_iter().map(|age_id| {
-                                html!("span", {
-                                    .text_signal(state.age_label(age_id))
-                                })
-                            }))
+                            // .child_signal(jig.age_ranges.signal_cloned().map(|ages_hash| {
+                            //     ages_hash.into_iter().map(|age_id| {
+                            //         html!("span", {
+                            //             .text(state.age_label(age_id))
+                            //         })
+                            //     })
+                            // }))
```

Comment:
I can't seem to pass Mutable HashSets in as children. This may be an easy issue to fix